mods/ra/weapons/missiles.yaml

```diff
@@ -279,15 +279,15 @@ SubMissile:
 
 SubMissileAA:
 	Inherits: SubMissile
-	Range: 6c0
+	Range: 8c0
 	ValidTargets: Air
 	Projectile: Missile
-		Speed: 162
-		Inaccuracy: 0c938
-		HorizontalRateOfTurn: 1
-		RangeLimit: 7c0
+		Speed: 234
+		Inaccuracy: 0c614
+		HorizontalRateOfTurn: 15
+		RangeLimit: 9c0
 	Warhead@1Dam: SpreadDamage
-		Damage: 20
+		Damage: 15
 		ValidTargets: Air
```

Comment:
Not in this case. When a missile is fired against a target in air who then lands, the missile will deliver no damage.